jcl/src/java.base/share/classes/java/lang/Access.java

```diff
@@ -363,6 +363,7 @@ public List<Method> getDeclaredPublicMethods(Class<?> clz, String name, Class<?>
 		return clz.getDeclaredPublicMethods(name, types);
 	}
 
+	/*[IF (JAVA_SPEC_VERSION < 25) | INLINE-TYPES]*/
 	/*[IF JAVA_SPEC_VERSION >= 15]*/
```

Comment:
Rather than add another layer of conditions, suggest inverting this and using `ELSEIF`:
```java
	/*[IF JAVA_SPEC_VERSION < 15]*/
	public void addOpensToAllUnnamed(Module fromModule, Iterator<String> packages) {
		fromModule.implAddOpensToAllUnnamed(packages);
	}
	/*[ELSEIF (JAVA_SPEC_VERSION < 25) | INLINE-TYPES]*/
	public void addOpensToAllUnnamed(Module fromModule, Set<String> concealedPackages, Set<String> exportedPackages) {
		fromModule.implAddOpensToAllUnnamed(concealedPackages, exportedPackages);
	}
	/*[ENDIF] JAVA_SPEC_VERSION < 15 */
```

Reply:
Updated accordingly